test/test_tensorexpr.py

```diff
@@ -1021,9 +1034,10 @@ def test_dynamic_shape():
         def test(x, y, z):
             return x * y * z
         cuda = CudaCodeGenCreated()
-        x, y, z = [torch.rand(4, 8).cuda() for _ in range(3)]
-        ref = test(x, y, z)
-        _ = test(*[torch.rand(6, 8).cuda() for _ in range(3)])
-        res = test(x, y, z)
+        with existing_fuser_disabled():
```

Comment:
it is better to put this under a "decorator". 

A better way to put all our tests under a fixture, which will initialize the tests.